roles/KubevirtNodeLabeller/defaults/main.yml

```diff
@@ -2,12 +2,7 @@
 # defaults file for KubevirtNodeLabeller
 wait: true
 use_kvm: true
-
-kubevirt_node_labeller_image: "node-labeller"
-kubevirt_node_labeller_tag: "latest"
-kvm_info_nfd_plugin_image: "kvm-info-nfd-plugin"
-kvm_info_nfd_plugin_tag: "latest"
-kubevirt_cpu_nfd_plugin_image: "cpu-nfd-plugin"
-kubevirt_cpu_nfd_plugin_tag: "latest"
-virt_launcher_image: "virt-launcher"
-virt_launcher_tag: "latest"
+kubevirt_node_labeller_image: kubevirt-cpu-node-labeller
```

Comment:
Why you add kubevirt- prefix to all image names? It will break whole node-labeller role, because these images don't exists on kubevirt quay.